docs/FAQ.md

```diff
@@ -68,13 +68,13 @@ Be careful as this is a destructive operation.
 Which is usually: `~/Library/Caches/Homebrew`
 
 ## My Mac `.app`s don’t find `/usr/local/bin` utilities!
-GUI apps on macOS don’t have `/usr/local/bin` in their `PATH` by default.
-If you’re on Mountain Lion, you can fix this by running
-`launchctl setenv PATH "/usr/local/bin:$PATH"`. [More details
-here](https://stackoverflow.com/questions/135688/setting-environment-variables-in-os-x/5444960#5444960),
-including how to set this across reboots. If you’re pre-Mountain Lion,
-[here’s an official
-alternative](https://developer.apple.com/legacy/library/qa/qa1067/_index.html).
+GUI apps on macOS don’t have `/usr/local/bin` in their `PATH` by
+default.  If you're on Mountain Lion or later, you can fix this by
+running `sudo launchctl config user path "/usr/local/bin:$PATH"` and
+then rebooting. Note that this sets the launchctl PATH for _all users_
+([more details here](https://apple.stackexchange.com/a/198282)). For
```

Comment:
Unfortunately, it looks like the sole source of documentation for this is in the man page for `launchctl`, and it appears that Apple doesn't put those online anymore: https://forums.developer.apple.com/thread/46067 .

However, it's easy enough to just refer to the manpage. Did that.